cmd/immudb/fips/fips.go

```diff
@@ -0,0 +1,30 @@
+//go:build fips
+// +build fips
+
+/*
+Copyright 2022 Codenotary Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	_ "crypto/tls/fipsonly"
+
+	immudb "github.com/codenotary/immudb/cmd/immudb/command"
+)
+
+func main() {
+	immudb.Execute()
```

Comment:
@tomekkolo this is the only place a log can be added to mention if it is running in FIPS mode

Reply:
fixed